src/commonMain/kotlin/komem/litmus/AutoOutcome.kt

```diff
@@ -0,0 +1,29 @@
+package komem.litmus
+
+interface AutoOutcome {
+    fun getOutcome(): Any?
+}
```

Comment:
What is the purpose of `AutoOutcome` interface? 
What `Auto` prefix stands for?